pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -184,25 +185,26 @@ private static String toErrorMessage(String table, String column, Object value,
         return String.format("Error updating %s.%s with value %s for %s=%s", table, column, value, primaryKey, id);
     }
 
-    static boolean isOncHistoryPatch(Patch originalPatch) {
+    static boolean isOncHistoryDeletePatch(Patch originalPatch) {
         return originalPatch.getNameValue().getName().equals(ONC_HISTORY_DELETED_FIELD) &&
                 originalPatch.getTableAlias().equals(DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS);
     }
 
-    static boolean isTissuePatch(Patch originalPatch) {
+    static boolean isTissueDeletePatch(Patch originalPatch) {
         return originalPatch.getNameValue().getName().equals(TISSUE_DELETED_FIELD) &&
                 originalPatch.getTableAlias().equals(DBConstants.DDP_TISSUE_ALIAS);
     }
 
     //creates a list of patches for deleting tissues that belong to an onc history
     protected static List<Patch> getPatchForTissues(Patch oncHistoryPatch) {
-        List<String> tissueIds = TissueDao.getTissuesByOncHistoryDetailId(oncHistoryPatch.getId());
+        List<Integer> tissueIds = TissueDao.getTissuesByOncHistoryDetailId(Integer.parseInt(oncHistoryPatch.getId()));
```

Comment:
Consider adding a `getIdAsInt()` method to `Patch`, and then calling that when you need the int instead of the string.  This will facilitate code re-use and relieve callers of the burden of casting/parsing.